package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@opentermsarchive/federated-api",
+  "version": "0.0.0",
+  "homepage": "https://github.com/OpenTermsArchive/federated-api#readme",
+  "bugs": {
+    "url": "https://github.com/OpenTermsArchive/federated-api/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OpenTermsArchive/federated-api.git"
+  },
+  "license": "EUPL-1.2",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint .",
+    "start": "node src/index.js",
+    "test": "cross-env NODE_ENV=test mocha --recursive \"./src/**/*.test.js\" --exit"
```

`lint` has been added as a script, but is not run in `test`, thus is likely to not be run locally and is at this stage not run in CI.
Either this:
```suggestion
    "test:unit": "cross-env NODE_ENV=test mocha --recursive \"./src/**/*.test.js\" --exit",
    "test": "npm run lint && npm run test:unit"
```
or at least add `npm run lint` in the `test` CI workflow 🙂